### PR TITLE
First pull request & Code review

### PR DESCRIPTION
**Pull request by grauJavier**
I have followed each of the requirements listed in the '[Hello Microverse Project](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)' Which finishes opening a **pull request** and asking for a code review.

### CODE REVIEW

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] Kindly please make sure to use the readme template provided to you by microverse, this is a requirement for every project you will work on all through your time in microverse [link to readme template](https://github.com/microverseinc/readme-template) :+1: 
readme
 
- [ ] when making use of this template it is worthy to note that you are required to customise it fully with all of your information as well as information about the project you worked on  providing all of the necessary links and license file, make sure no link is broken before you resubmit. :+1: 
- [ ] this is an example of a students readme for this same project you can make use of it as a template as well remember to cinfigure it with all of your information. [try this example template](https://github.com/HFG43/Hello-Microverse-/blob/main/README.md) :+1:

Comment (Efiamotu-1) [inline index.html:1]: - [ ] linters are as well important for your project setup as you would use them throughout your lifetime in the microverse program so it is best you start learning how to include them in your projects. follow the step by step guide in this link [linters link](https://github.com/microverseinc/linters-config/tree/master/html-css) and you will be able to setup linters correctly for your next PR. :+1:

Comment (Efiamotu-1) [inline index.html:1]: - [ ] It is also important to use good PR title and summary when making a pull request attached below is an image of how your pr title and summary for this project should look like please make sure to make the correct changes before resubmitting for a new pull request. :+1: 
- [ ] Make sure to also name your branches descriptively tailored to the changes you would be making for example `add-homepage`, `mobile-menu-feature` etc
good-pr

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @grauJavier , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## STATUS: CHANGES REQUIRED :recycle: 


## Highlights :dart: 
* Good job implementing some of the projects requirements :+1: 
* Good commit message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (grauJavier) [reply index.html:1]: Hi @Efiamotu-1! 
Thank you very much for your review. Momentarily one thing is not 100% clear to me. Should I have a branch for each file? (HTML, CSS, test.md) or is it fine to have it in one branch?

Comment (Efiamotu-1) [reply index.html:1]: @grauJavier you only need a single branch and you would add all the needed files for that milestone in it that's all. does this answeer your question?

**Pull request closed without merging**